lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -21,146 +22,131 @@ describe('NxRadio', function() {
     children: undefined
   };
 
-  const getShallow = getShallowComponent<Props>(NxRadio, simpleProps);
+  const renderEl = rtlRenderElement<Props>(NxRadio, minimalProps);
+  const quickRender = rtlRender<Props>(NxRadio, minimalProps);
 
   it('renders a <label> containing a radio <input> and .nx-radio__circle and .nx-radio__outer-circle  elements',
       function() {
-        const shallowRender = getShallow();
+        const el = renderEl()!;
 
-        expect(shallowRender).toMatchSelector('label.nx-radio');
+        expect(el.tagName).toBe('LABEL');
+        expect(el).toHaveClass('nx-radio');
 
-        expect(shallowRender.find('input')).toHaveProp('name', 'color');
-        expect(shallowRender.find('input')).toHaveProp('type', 'radio');
-        expect(shallowRender.find('input')).toHaveProp('disabled', false);
-        expect(shallowRender.find('input')).toHaveProp('checked', false);
+        expect(el.querySelector('input')).toHaveAttribute('name', 'color');
+        expect(el.querySelector('input')).toHaveAttribute('type', 'radio');
+        expect(el.querySelector('input')).not.toHaveAttribute('disabled');
+        expect(el.querySelector('input')).not.toHaveAttribute('checked');
 
-        expect(shallowRender.find('input')).toHaveClassName('nx-radio__input');
+        expect(el.querySelector('input')).toHaveClass('nx-radio__input');
 
-        expect(shallowRender.find('svg.nx-radio__circle')).toExist();
-        expect(shallowRender.find('.nx-radio__circle .nx-radio__outer-circle')).toExist();
+        expect(el.querySelector('svg')).toHaveClass('nx-radio__circle');
+        expect(el.querySelector('circle')).toHaveClass('nx-radio__outer-circle');
       }
   );
 
   it('adds classes specified with the className prop', function() {
-    const component = getShallow({ className: 'foo' });
+    const component = renderEl({ className: 'foo' });
 
-    expect(component).toHaveClassName('foo');
-    expect(component).toHaveClassName('nx-radio');
+    expect(component).toHaveClass('foo');
+    expect(component).toHaveClass('nx-radio');
```

Comment:
This should use the `for` loop pattern we've established in other RTL tests

Reply:
> This should use the `for` loop pattern we've established in other RTL tests

Updated her: 984bdacd64b861440c86635dd61de604207c7a83